e2e/test/E2ETests.csproj

```diff
@@ -121,5 +125,17 @@
     <PackageReference Include="Microsoft.Azure.Devices.Provisioning.Service" Version="1.*" />
     <ProjectReference Include="$(RootDir)\security\tpm\samples\SecurityProviderTpmSimulator\SecurityProviderTpmSimulator.csproj" />
   </ItemGroup>
-
+  <ItemGroup Condition="'$(TargetFramework)' == 'net47'">
+    <PackageReference Include="Microsoft.Azure.OperationalInsights">
+      <Version>0.10.0-preview</Version>
```

Comment:
We shouldn't use `preview` NuGet releases in the master branch. Is there a release version of this package?

Reply:
I believe I can remove the preview nuget.  Thanks.